api/swagger.yaml

```diff
@@ -2015,6 +2015,56 @@ definitions:
             description: "A list of additional groups that the container process will run as."
             items:
               type: "string"
+          Privileges:
+            type: "object"
+            description: "Security options for the container"
+            properties:
+              CredentialSpec:
+                type: "object"
+                description: "CredentialSpec for managed service account (Windows only)"
+                properties:
+                  File:
+                    type: "string"
+                    description: |
+                      Load credential spec from this file. The file is read by the daemon, and must be present in the
+                      `CredentialSpecs` subdirectory in the docker data directory, which defaults to
+                      `C:\ProgramFiles\Docker\` on Windows.
+
+                      For example, specifying `spec.json` loads `C:\ProgramFiles\Docker\CredentialSpecs\spec.json`.
+
+                      <p><br /></p>
```

Comment:
Sorry, there are a bit dirty, but I couldn't find another way to add some spacing above the `note`